src/applications/vaos/appointment-list/components/RequestedAppointmentDetailsPage.jsx

```diff
@@ -94,9 +94,9 @@ function RequestedAppointmentDetailsPage({
 
   return (
     <div>
-      <div className="vads-u-display--block vads-u-padding-y--2p5">
-        ‹ <Link to="/requested">Manage appointments</Link>
-      </div>
+      <Breadcrumbs>
+        <Link to="/">Appointment detail</Link>
```

Comment:
Similar to above, maybe "Request detail"

Reply:
Good catch. This makes total sense.